virt/virt-4-9-release-notes.adoc

```diff
@@ -45,6 +45,7 @@ Red Hat is committed to replacing problematic language in our code, documentatio
 //CNV-12858 SVVP previous 8914
 
 //CNV-9540 High performance windows
+High-performance templates are now available for Windows servers.
```

Comment:
FYI, the link in the PR description points to the incorrect Jira story. After looking at the epic associated with [CNV-9540](https://issues.redhat.com/browse/CNV-9540), it seems that the high-performance templates are available for both Windows 10 and the Windows Server operating systems.

I would suggest rewording your release note to something like this: "High-performance virtual machine templates are now available for supported Windows operating systems."

The supported guest operating systems are listed in a separate section in the release notes. You could link to that section. Please check with QE for accuracy.

Reply:
Thanks, Shika! (link fixed)
@kbidarkar Please review the suggested revision for accuracy. 
 "High-performance virtual machine templates are now available for supported Windows operating systems."

Reply:
Yes, wording in the above comment looks more accurate now, as the command `oc get templates -n openshift | grep -i windows.*-highperformance` lists both windows-10 and windows servers.